app/src/test/java/com/teamagam/gimelgimel/app/model/ViewsModels/MessageJsonAdapterTest.java

```diff
@@ -25,8 +26,10 @@ public void checkMessageTypeJsonConverter() {
 
         //Arrange
         String senderId = "sender1";
+        LocationEntity location = new LocationEntity(new PointGeometry(23, 32), "example", LocationEntity.REGULAR);
+
         Message msgT = new MessageText(senderId, "text123");
-        Message msgL = new MessageLatLong(senderId, new PointGeometry(23, 32));
+        Message msgL = new MessageGeo(senderId, location);
```

Comment:
Im guessing the original author of this method meant that msgL means MessageLongLat, which is now MessageGeo. 
So it would probably make more sense to name is msgG, but I would take the opportunity and rename all of these local variables into more informative names. messageText, messageGeo, ,messageImage
!fix